app/controllers/api/channels_controller.rb

```diff
@@ -21,8 +21,11 @@ def create
             "page_id" => "", "verify_token" => SecureRandom.base58, "access_token" => ""
           }
         when 'websocket'
+          access_token = SecureRandom.uuid
+
           bot.channels.create! kind: "websocket", name: "Web", config: {
-            "access_token" => SecureRandom.uuid
+            "access_token" => access_token,
+            "url_key" => bot.uuid ? Shortener::ShortenedUrl.generate("/c/#{bot.uuid}/#{access_token}").unique_key : ""
```

Comment:
The downside of having this short link as part of the channel config is that you need to remember to delete the short link when deleting the channel. Maybe having a foreign key with a belongs_to and let rails take care of it? It's either that or adding a `before_destroy` method that deletes the links.